packages/app-store/amie/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "/*": "Don't modify slug - If required, do it using cli edit command",
+  "name": "Amie",
+  "slug": "amie",
+  "type": "amie_other",
+  "imageSrc": "/api/app-store/amie/icon.svg",
+  "logo": "/api/app-store/amie/icon.svg",
+  "url": "https://cal.com/apps/amie",
+  "variant": "other",
+  "categories": ["calendar"],
```

Comment:
@hariombalhara i did the same as with vimcal but it doesnt show up in "calendar" categories